Documentation/component-description.md

```diff
@@ -84,8 +84,8 @@ In this page, we focus on the status of each elements. For requirements and desi
 - [tensor\_mux](https://github.com/nnstreamer/nnstreamer/tree/main/gst/nnstreamer/tensor_mux) (stable)
 - [tensor\_demux](https://github.com/nnstreamer/nnstreamer/tree/main/gst/nnstreamer/tensor_demux) (stable)
 - [tensor\_source](https://github.com/nnstreamer/nnstreamer/tree/main/gst/nnstreamer/tensor_source) (stable for IIO. More sources coming soon)
-- [tensor\_aggregator](https://github.com/nnstreamer/nnstreamer/tree/main//gst/nnstreamer/tensor_aggregator) (stable)
-- [tensor\_repo\_sink](https://github.com/nnstreamer/nnstreamer/tree/main/gst/nnstreamer/tensor_repo) (stable)
+- [tensor\_aggregator](https://github.com/nnstreamer/nnstreamer/tree/main/gst/nnstreamer/tensor_aggregator) (stable)
+- [tensor\_repo\_sink](https://github.com/nnstreamer/nnstreamer/tree/main/gst/nnstreamer/tensor_sink) (stable)
```

Comment:
nice catch

Reply:
Gichan told me that "tensor_repo" is correct :(